gpytorch/lazy/lazy_evaluated_kernel_tensor.py

```diff
@@ -103,13 +141,16 @@ def evaluate_kernel(self):
                 x1 = self.x1
                 x2 = self.x2
 
-            self._cached_kernel_eval = super(Kernel, self.kernel).__call__(x1, x2, **self.params)
+            self._cached_kernel_eval = super(Kernel, self.kernel).__call__(
+                x1, x2, diag=False, batch_dims=self.batch_dims, **self.params
+            )
             if self.squeeze_row:
-                self._cached_kernel_eval.squeeze_(-2)
+                self._cached_kernel_eval = self._cached_kernel_eval.squeeze(-2)
```

Comment:
Did the in-place operations cause issues with autograd or something?

Reply:
Oh I think this may have been left over from some crazy debugging... I can revert it